src/pages/LandingPanel.vue

```diff
@@ -0,0 +1,20 @@
+<script>
+import { defineComponent } from 'vue';
+
+export default defineComponent({});
+</script>
+
+<template>
+  <main class="landing-panel">
+    <router-view />
+  </main>
+</template>
+
+<style lang="scss">
+.landing-panel {
+  display: flex;
+  flex-direction: column;
+  gap: 6.4rem;
+  // min-height: calc(100vh + 1px);
```

Comment:
I used this just to make sure the page is scrollable when I had not enough content for it to be so.
It is of course to be deleted 😅